e2e/app.e2e.ts

```diff
@@ -2,6 +2,7 @@ const Application = require('spectron').Application;
 const { resolve } = require('path');
 import * as os from 'os';
 import { expect } from 'chai';
+import * as clone from 'clone';
```

Comment:
we don't need any extra dependencies imho. if you want to create multiple instances then export function inside `beforeEach` and call this function to create new window. this is overkill. 
hope you didn't struggling with that since Wednesday, because I cannot see any progress then ~5 falsy tests.